cirq/contrib/qasm_import/_parser_test.py

```diff
@@ -151,3 +152,108 @@ def test_syntax_error():
 
     with pytest.raises(QasmException, match=r"""Syntax error: 'bla'.*"""):
         parser.parse(qasm)
+
+
+def test_CX_gate():
+    qasm = """
+     OPENQASM 2.0;          
+     qreg q1[2];
+     qreg q2[2];
+     CX q1[0], q1[1];
+     CX q1, q2[0];
+     CX q2, q1;      
+"""
+    parser = QasmParser()
+
+    q1_0 = cirq.NamedQubit('q1_0')
+    q1_1 = cirq.NamedQubit('q1_1')
+    q2_0 = cirq.NamedQubit('q2_0')
+    q2_1 = cirq.NamedQubit('q2_1')
+
+    expected_circuit = Circuit()
+    # CX q1[0], q1[1];
+    expected_circuit.append(cirq.CNOT(q1_0, q1_1))
+    # CX q1, q2[0];
+    expected_circuit.append(cirq.CNOT(q1_0, q2_0))
+    expected_circuit.append(cirq.CNOT(q1_1, q2_0))
+    # CX q2, q1;
+    expected_circuit.append(cirq.CNOT(q2_0, q1_0))
+    expected_circuit.append(cirq.CNOT(q2_1, q1_1))
+
+    parsed_qasm = parser.parse(qasm)
+
+    assert parsed_qasm.supportedFormat is True
+    assert parsed_qasm.qelib1Include is False
+
+    ct.assert_same_circuits(parsed_qasm.circuit, expected_circuit)
+    assert parsed_qasm.qregs == {'q1': 2, 'q2': 2}
+
+    cirq.Simulator().simulate(parsed_qasm.circuit)
```

Comment:
Why the call to the simulator if we are not using the results from it?

Reply:
I know ... it looks weird. I had this in because at one point as I was experimenting, I had a version where I returned sympy numbers instead of actual floats and even though circuit comparison succeeded - but the circuits were not "simulatable" - I wanted to guard against that, but probably this is not the right place to do so (I created an issue a while ago regarding that case https://github.com/quantumlib/Cirq/issues/1611). 
I took out this "non-assertion check" from here.